granulate_utils/metadata/databricks_client.py

```diff
@@ -34,6 +36,11 @@ def __init__(self, logger: logging.LoggerAdapter) -> None:
         else:
             self.logger.debug(f"Got Databricks job name: {self.job_name}")
 
+    def _request_get(self, url: str) -> requests.Response:
+        resp = requests.get(url, timeout=REQUEST_TIMEOUT)
+        resp.raise_for_status()
+        return resp
+
     @staticmethod
     def get_webui_address() -> Optional[str]:
```

Comment:
Not `Optional` anymore. Or rather, never was...

Reply:
c53ddb6